src/graph_export/graph_cli_parser.cpp

```diff
@@ -125,12 +121,35 @@ void GraphCLIParser::prepare(ServerSettingsImpl* serverSettings, ModelsSettingsI
         // Pull with default arguments - no arguments from user
         if (serverSettings->hfSettings.pullHfModelMode) {
```

Comment:
we shouldn't go here anyway if pullHfMode is not specified so this if is not required here. Again use only graphSettings in this method.

Reply:
We will also start ovms and pull models, grate graphs in non pull mode.